### DISCUSSION TITLE
Budgie Unresponsive upon resume from Automatic Suspend

### DISCUSSION THREAD

**Discussion by Spaceboy60** [SpiralLinux/SpiralLinux-project#506] [General]
OK, this doesn't happen all the time, however it has happened 3 times in the last week.  System completely unresponsive upon resumption, only option is to go to TTY and do a "sudo reboot".  journalctl -b below:

https://pastecode.io/s/p9wkfd2v

Any suggestions??

**Comment by geckolinux** [comment]
Hi there, it's probably a buggy ACPI (power control) implementation in your computer's BIOS and the Linux kernel needs to include a workaround for it. This is way outside of the scope of what SpiralLinux can control, so you can file a detailed Debian bug report.

**Reply by Spaceboy60** [reply]
> Hi there, it's probably a buggy ACPI (power control) implementation in your computer's BIOS and the Linux kernel needs to include a workaround for it. This is way outside of the scope of what SpiralLinux can control, so you can file a detailed Debian bug report.

If that's the case how comes it wasn't doing it the first month I was using Spiral. Only in the last couple of weeks after doing some updates. Also when I pasted Journalctl a couple of weeks ago to reddit after the issue occurred, a user responded saying it was a kernel l crash that had caused the issue according to the logs I posted at the time. 

Not saying this is also kernel related, I don't know what I'm looking for in journalctl, that's why I posted it here

**Reply by geckolinux** [reply]
Possibly a kernel update since you installed SpiralLinux? You could try other kernel versions that you have installed and/or install other versions that are available, search for `linux-image-6` in Synaptic. At any rate it's an upstream issue that is not specific to SpiralLinux.
